src/pages/index.js

```diff
@@ -69,15 +37,59 @@ const profileDescriptionInput = document.querySelector(
   "#profile-description-input"
 );
 
-// Cards
-const addCardModal = document.querySelector("#add-card-modal");
+// ********************************* API ******************************** //
 
-// Modals
-const profileEditForm = profileEditModal.querySelector(".modal__form");
-const addButtonForm = addCardModal.querySelector(".modal__form");
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "16b910f9-e28d-483c-ac51-2ec35ebc8735",
+    "Content-Type": "application/json",
+  },
+});
+
+let userInfoInstance;
+
+// Initialiize, Generate User Profile and Cards on Load
+api
+  .getUserInfo()
+  .then((userInfo) => {
+    const { name, about, avatar } = userInfo;
+    userInfoInstance = new UserInfo(
+      profileTitle,
+      profileDescription,
+      avatarImg
+    );
+
+    userInfoInstance.setUserInfo({
+      name,
+      job: about,
+      avatar: avatar,
+    });
+    api.getInitialCards().then((res) => {
```

Comment:
**Could be improved**

You need to combine the requests of the user's info and the cards together in `Promise.all` not to make **nested** api calls
 
More info is here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

```js

Promise.all([getUserInfo(), getCards()])
// destructure the response
  .then(([userData, cards]) => {
      // set all the data
  })
  .catch(err => {
    // catch possible errors
  });

```